warehouse/query-core/src/main/java/datawave/query/cardinality/CardinalityRecord.java

```diff
@@ -222,8 +222,7 @@ public static CardinalityRecord readFromDisk(File file) {
         
         CardinalityRecord cardinalityRecord = null;
         ObjectInputStream ois = null;
-        try {
-            FileInputStream fis = new FileInputStream(file);
+        try (FileInputStream fis = new FileInputStream(file)) {
             ois = new ObjectInputStream(fis);
```

Comment:
It looks like ois can be moved into the try and then remove the IOUtils.closeQuietly call below.

Reply:
Fixed in next commit